pkg/identity/identity.go

```diff
@@ -55,8 +55,8 @@ type MonitoredValues struct {
 	CustomExtensions []extensions.CustomExtension `yaml:"customExtensions"`
 }
 
-// RekorLogEntry holds a certificate subject, issuer, OID extension and associated value, and log entry metadata
-type RekorLogEntry struct {
+// LogEntry holds a certificate subject, issuer, OID extension and associated value, and log entry metadata
```

Comment:
Renamed RekorLogEntry to LogEntry to be used in both Rekor and CT log entries found via identity monitoring- would it be better to keep them as distinct structs?

Reply:
Let's reuse, it minimizes duplicated code.